src/common/Error/Index.js

```diff
@@ -0,0 +1,28 @@
+import { StyledNavLink } from "../Header/Navigation/styled";
+import { Wrapper, ErrorIcon, Title, Subtitle, Button } from "./styled";
+import { MainWrapper } from "../MainWrapper";
+
+export const Error = () => {
+  return (
+    <MainWrapper
+      content={
+        <Wrapper>
+          <ErrorIcon />
+          <Title>
+            Ooops! Something went wrong...
+          </Title>
+          <Subtitle>
+            Please check your network connection and try again
+          </Subtitle>
+          <StyledNavLink
+            to="/movies"
+          >
+            <Button>
+              Back to home page
+            </Button>
+          </StyledNavLink>
+        </Wrapper >
+      }
+    />
+  );
+};
```

Comment:
And i forgot to mention, that you must name this file with a lowercase letter: `index.js` except of `Index.js`. Otherwise it will be a fatal error